languages/en.json

```diff
@@ -132,6 +132,45 @@
   "MYTHACRI.MayhemUpdated": "Updated your mayhem points to ({points}).",
   "MYTHACRI.Pips": "Pips",
   "MYTHACRI.Refresh": "Refresh",
+  "MYTHACRI.REGION": {
+    "TRAP": {
+      "FIELDS": {
```

Comment:
Not used directly in the module, this is a native feature of Foundry v12, localizing any data fields automatically.